From d06dd0d474c14cbe0a4c28386a1c303a3b42aca7 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Fri, 20 Dec 2019 15:07:05 +0900
Subject: [PATCH 5/6] docs: add desc for env command

This update is to add description for env command in `Common Commands`
section.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 docs/guides/commands/common.rst | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/docs/guides/commands/common.rst b/docs/guides/commands/common.rst
index cfa4eaf..d3b8262 100644
--- a/docs/guides/commands/common.rst
+++ b/docs/guides/commands/common.rst
@@ -230,6 +230,23 @@ You cannot unregister node under the control, or switch to other one before.
     Cannot del server "1" in use!
 
 
+.. _commands_common_env:
+
+env
+---
+
+Show environmental variables. It is mainly used to find variables related to
+SPP.
+
+.. code-block:: console
+
+    # show all env varibles.
+    spp > env
+
+    # show env varibles starts with `SPP`.
+    spp > env SPP
+
+
 .. _commands_common_pwd:
 
 pwd
-- 
2.17.1

